spec/spec_helper_acceptance.rb

```diff
@@ -15,6 +15,7 @@
   end
 end
 
+$parallel = { :run_in_parallel => ENV['BEAKER_SIMP_parallel'] =~ /y|true/ }
```

Comment:
This should just be a regular variable instead of a constant. Doesn't matter a huge amount but should not be carried through as a pattern.

Reply:
Just tried this and would like to suggest the following:

`parallel = { :run_in_parallel => ['yes','true','on'].include?(ENV['BEAKER_SIMP_parallel']) }`

Reply:
@trevor-vaughan Great minds think alike: https://github.com/simp/pupmod-simp-tpm2/blob/master/spec/acceptance/suites/default/00_init_spec.rb#L4-L5:

```ruby
# before tests
RUN_IN_PARALLEL = ENV.fetch('BEAKER_RUN_IN_PARALLEL', '').split(',').include?('tests')

# in tests
apply_manifest_on(hosts, manifest, run_in_parallel: RUN_IN_PARALLEL)
```